libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Utils.java

```diff
@@ -360,20 +362,25 @@ public static String fieldValueAsString(Object fieldValue) {
             return nullToEmpty((String) fieldValue);
         }
         if (fieldValue instanceof Map) {
-            List<String> mapEntriesAsStrings = nullToEmptyMap(((Map<String, Object>) fieldValue)).entrySet().stream()
-                    .map(e -> {
-                        String valueString = e.getValue() != null ? e.getValue().toString():"";
-                        return e.getKey() + " : " + valueString;
-                    })
-                    .collect(Collectors.toList());
-            return joinStrings(mapEntriesAsStrings);
+            Map<String, Object> originalMap = nullToEmptyMap(((Map<String, Object>) fieldValue));
+            Map<String, String> map = Maps.transformValues(originalMap, v -> v != null ? v.toString() : "");
+            return serializeToJson(map);
         }
         if (fieldValue instanceof Iterable){
-            return joinStrings((Iterable<String>) fieldValue);
+            return serializeToJson(fieldValue);
         }
         return fieldValue.toString();
     }
 
+    private static String serializeToJson(Object value) {
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            return mapper.writeValueAsString(value);
+        } catch (JsonProcessingException e) {
+            throw new IllegalArgumentException(String.format("Cannot serialize field value %s to JSON", value), e);
```

Comment:
It might be better to use no `RuntimeException` here, maybe use instead `SW360Exception`.